docs/querying/sql-window-functions.md

```diff
@@ -0,0 +1,164 @@
+---
+id: sql-window-functions
+title: Window functions
+---
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one
+  ~ or more contributor license agreements.  See the NOTICE file
+  ~ distributed with this work for additional information
+  ~ regarding copyright ownership.  The ASF licenses this file
+  ~ to you under the Apache License, Version 2.0 (the
+  ~ License); you may not use this file except in compliance
+  ~ with the License.  You may obtain a copy of the License at
+  ~
+  ~   http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ AS IS BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+> Apache Druid supports two query languages: [Druid SQL](sql.md) and [native queries](querying.md).
+> This document describes the SQL language.
+>
+> Window functions are an [experimental](../development/experimental.md) feature. Development and testing are still at early stage. Feel free to try window functions and provide your feedback.
+> There are known issues where ORDER BY only works on ascending order and certain options may cause errors.
+> Set the context parameter `windowsAreForClosers: true` to use window functions.
```

Comment:
Instead of having this in a note, could this go into a Prerequisites section?

Reply:
It's not exactly a pre-requisite. You do it per-query. And it's like a feature flag that will go away once the feature is AF

Reply:
```suggestion
> To use window functions, submit the context parameter `windowsAreForClosers: true` with your query.
```

Reply:
Maybe also link to context parameters in case someone doesn't know how to set it?